### DISCUSSION TITLE
Zotero-Obsidian sync not working anymore?

### DISCUSSION THREAD

**Discussion by aaajjjsss** [windingwind/zotero-better-notes#1001] [Q&A]
Hi, I just downloaded Zotero 7 yesterday and was able to set up the two-way sync between Zotero and Obsidian using Better Notes and BetterBibTeX plugins. Today I can't sync notes from Zotero into Obsidian using the Better Notes export or sync functions. When I open the sync manager, I can see that the markdown path is set to the right folder in Obsidian, but the note doesn't show up. It works when I used the regular Zotero export note function.

Any thoughts? Thank you!

**Comment by aaajjjsss** [comment]
Forgot to mention - in the Better Notes "export" menu, the "BetterNotes-export-title" window that pops up has an incorrect path under "MarkDown Path." I'm wondering if this is the issue, but I don't know how to change the path. As mentioned above, the sync manager path is correct.

**Comment by windingwind** [comment]
If you have any bug (or you think it might be a bug), please report it as an issue, with all the required details. I can't help you further here and will close this discussion.
